src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -246,7 +248,10 @@ functionsToRunOnDataTablesDrawEvent: [],
           searching: @json($crud->getOperationSetting('searchableTable') ?? true),
           ajax: {
               "url": "{!! url($crud->route.'/search').'?'.Request::getQueryString() !!}",
-              "type": "POST"
+              "type": "POST",
+              "data": {
+                "unfilteredQueryCount": "{{$crud->getOperationSetting('unfilteredQueryCount') ?? false}}"
+            },
```

Comment:
Do we actually need to include `unfilteredQueryCount` as a parameter upon each search request? I don't understand why...  I think this is only needed if we let the user change the count on the front-end... which... we don't... right?

Otherwise, `unfilteredQueryCount` is in our control in the endpoint too... so why not define true/false only there?

Reply:
When the search ajax query is performed we already have the page loaded so the query to get the `unfilteredQueryCount` already had run.
If we send it  for the ajax endpoint we don't need to "re-do" this `unfilteredQueryCount` and only do the `filteredQueryCount` after applying what was sent to the `search()` endpoint, otherwise we would calculate the `filteredQueryCount` when defining the panel filters

Note that: `adding filters to the crud applies the query constrain` so we need to get the `unfilteredQuery` before applying any filter or any seach, only account for the base $crud->query after the `setups` definitions in the CrudController.